rubocop.yml

```diff
@@ -106,3 +108,13 @@ Style/SignalException:
 
 Layout/SpaceAroundOperators:
   AllowForAlignment: true
+
+Metrics/ModuleLength:
+  Exclude:
+    - "**/*_spec.rb"
+
+RSpec/MultipleExpectations:
+  Max: 10
```

Comment:
I think if you allow 10 expectations, you might as well disable this cop completely (no limit).